client/components/site-selector/index.jsx

```diff
@@ -78,10 +79,16 @@ export default React.createClass( {
 		this.refs.siteSearch.blur();
 	},
 
+	recordAddNewSite( user ) {
+		analytics.tracks.recordEvent( 'calypso_add_new_wordpress_click', {
+			'existing_sites': user.data.site_count
```

Comment:
Two small things here:

1. The `existing_sites` property doesn't need quotes around it because it's a valid JavaScript identifier. If it included spaces or dashes, for example, then you would need the quotes, but here you don't.

2. The analytics module automatically records a few additional properties about the event including `site_count` which should make recording this property unnecessary (we refer to them as [super properties](https://github.com/Automattic/wp-calypso/blob/master/client/analytics/super-props.js)). Once you do that, you can also get rid of the `user` argument from this function as well as the `onClick` event handler.